tools/RAiDER/losreader.py

```diff
@@ -725,4 +725,65 @@ def get_orbit(orbit_file, ref_time, pad):
 
     orb = isce.core.Orbit(svs_i)
 
-    return orb
\ No newline at end of file
+    return orb
+
+
+def build_ray(model_zs, ht, xyz, LOS, MAX_TROPO_HEIGHT=50000):
```

Comment:
I would specify the default max_tropo_height to be the value we define in the constants module. I think it's called Zmax? You can import that constant and then use it as a default value.

Reply:
I would also rename `ht` and `xyz` to be more explicit if this is the model xyz, the points xyz, etc. At the moment I'm confused what `ht` is. In the same vein definitely suggest to add a docstring with the args and return values, dtypes, and descriptions.

Reply:
@jlmaurer I only see Z_REF, which is 15000 and much lower than correct. it looks like this value is assigned to self._zmax in the weather models, but probably it's overwritten at some point? any thoughts on how best to proceed?

Reply:
This is accomplished in #560